api/src/main/resources/api/service_openapi.yaml

```diff
@@ -238,10 +238,16 @@ paths:
     parameters:
       - $ref: "#/components/parameters/UnderlayNameV2"
       - $ref: "#/components/parameters/EntityNameV2"
-    get:
```

Comment:
Is it difficult to create a new API? This seems like it's awkwardly shoehorned in here.

Reply:
Not difficult. I just struggled with how to best make this available in the API. I had another version where I put it in a separate endpoint, let me push that up and see if it seems cleaner to you.

Reply:
OK, got the alternate version working again. PTAL.

My main thought in implementing both of these alternatives was to have one way to lookup display hints. The hints might be across all instances (for demographics) or a subset defined by a related entity (for measurement detail). But the return information looks the same, so it seems like they should also be queryable in the same way.